teens_ui/src/router.js

```diff
@@ -7,6 +7,7 @@ Vue.use(Router)
 
 export default new Router({
   mode: 'history',
+  base: '/teens/',
```

Comment:
added base directory so that the router recognizes from where the routing directory starts